parsec/parsec.c

```diff
@@ -249,7 +249,7 @@ static void* __parsec_thread_init( __parsec_temporary_thread_initialization_t* s
     }
     gettimeofday(&tv_now, NULL);
 
-    PARSEC_TLS_SET_SPECIFIC(parsec_tls_execution_stream, es);
+    parsec_set_my_execution_stream(es);
```

Comment:
The only reason we need the `parsec_set_my_execution_stream` accessor is because parsec_tls_execution_stream is defined as static, and therefore not visible outside the parsec.c file. I think that where the symbols are visible, the uses of the TLS macros should be encouraged.

Reply:
Reverted